R/utils.R

```diff
@@ -200,3 +200,47 @@ ensure_data.table <- function(data) {
   }
   return(data)
 }
+
+#' @title Print information about a `forecast` object.
+#' @description This function prints information about a forecast object,
+#' including "Forecast type", "Score columns",
+#' "Forecast unit".
+#' @export
+#' @keywords print-forecast
+#' @examples
+#' print(example_quantile %>% as_forecast)
+print.forecast_binary <- function(dat, ...) {
+  # Obtain forecast object information for printing
+  forecast_type <- get_forecast_type(dat)
+  score_cols <- get_score_names(dat)
+  forecast_units <- get_forecast_unit(dat)
+
+  # Print forecast object information
+  cat("Forecast type:\n")
+  print(forecast_type)
+
+  if (!is.null(score_cols)) {
+    cat("\nScore columns:\n")
+    print(score_cols)
+  }
+
+  cat("\nForecast units:\n")
```

Comment:
The manuscript uses the singular, so I suggest we do it here as well: 
```suggestion
  cat("\nForecast unit:\n")
```